tests/unit/Core/Framework/Plugin/Composer/FactoryTest.php

```diff
@@ -24,7 +24,7 @@ public function testCreateComposer(): void
         $composer = Factory::createComposer(__DIR__ . '/../_fixtures/core');
 
         static::assertSame('shopware/platform', $composer->getPackage()->getName());
-        static::assertSame('6.6.9999999-dev', $composer->getPackage()->getVersion());
+        static::assertContains($composer->getPackage()->getVersion(), ['6.6.9999999-dev', '6.7.9999999.9999999-dev', 'dev-trunk']);
```

Comment:
Based on setup stuff, composer decides an different version, I added three variants so if it match one of them it's fine